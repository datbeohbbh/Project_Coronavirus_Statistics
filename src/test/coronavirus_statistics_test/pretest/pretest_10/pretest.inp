8 10
? 5
? 5
? 5
q 1 5 26
q 3 5 6
q 1 2 49
q 1 5 16
? 6
? 4 2
? 4 2