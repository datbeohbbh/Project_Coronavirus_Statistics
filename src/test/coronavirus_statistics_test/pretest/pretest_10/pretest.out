Query ? 5 :
The number of safe cities is 8

Query ? 5 :
The number of safe cities is 8

Query ? 5 :
The number of safe cities is 8

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 49
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 49
-> Has recovered : 0
-> Was dead : 0

